node/node.go

```diff
@@ -428,7 +429,12 @@ func (nn nexNode) IsTargetNode(inId string) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	if inId == pub || nn.auctionMap.Exists(inId) {
+	if inId == pub {
+		return true, nil
+	}
+	if nn.auctionMap.Exists(inId) {
+		auctionId := nn.auctionMap.Get(inId)
+		nn.options.Logger.Debug("Accepting auction", slog.String("auctionId", auctionId))
```

Comment:
nit: should be something like "accepting workload from auction" , otherwise this looks like the first step in an auction not the 2nd